components/scream/src/share/atm_process/atmosphere_process_group.cpp

```diff
@@ -155,6 +155,9 @@ void AtmosphereProcessGroup::run_sequential (const Real dt) {
 
   for (auto atm_proc : m_atm_processes) {
     // Run the process
+    //if (m_comm.am_i_root()) {
+    printf("Entering %s\n",atm_proc->name() );
+    //}
```

Comment:
@jeff-cohere - this is my attempt to add a log message to SCREAM (it doesn't seem to work in CIME runs and I commented out the i_am_root bit bit as a test and didn't bother to un-comment it). Would using your ekat logging just involve a single line change here, or is there more work to be done since SCREAM doesn't init the logger?

Reply:
EKAT has some custom logging tools, but you can also just use spdlog. The [README](https://github.com/gabime/spdlog) for that project has some good examples.

Tagging @bartgol in case he has thoughts.

Reply:
Technically, `atm_proc->name()` returns a C++ string, while `printf` only takes native C types, and `%s` is for C-style string (aka, a `char*`). I'm not sure if this is the issue (the compile might just pass the first part of a string object, which might as well be the char* stored inside), but it's worth pointing out. So the correct call should be
```
printf ("Entering %s\n", atm_proc->name().c_str());
```